src/diff/index.js

```diff
@@ -256,20 +256,9 @@ function diffElementNodes(dom, newVNode, oldVNode, context, isSvg, excessDomChil
 			excessDomChildren = EMPTY_ARR.slice.call(dom.childNodes);
 		}
 		if (newVNode!==oldVNode) {
-			let oldProps = oldVNode.props;
+			let oldProps = oldVNode.props || null;
```

Comment:
Hey @developit, as @JoviDeCroock noted, defaulting to `EMPTY_OBJ` would really get the size further down. Do you see any issues by reusing the `EMPTY_OBJ` ref instead of `null`?

Reply:
I'd prefer to split up the codepaths here so that attributes are diffed using a separate flow anyway. I'll check though, if the perf difference isn't huge then using an empty object here is fine.

Reply:
Awesome!

My concern would be that using `EMPTY_OBJ` would basically do the lookup on that object where with `null` that would not happen. So, I guess `null` could entail a bit of boost? But that's just by looking at it tbh. Gonna spin-up an bench to, hopefully, clear this.

Reply:
Alrighty! 😄 
So, using `EMPTY_OBJ` in my tests has no effect in terms of perf. The only thing would be the codepath split that you'd like to happen. Would this have an impact on splitting it up? I guess we would either do `oldProps === null` or `oldProps === EMPTY_OBJ`?